aa_modules/aamod_QA.m

```diff
@@ -0,0 +1,257 @@
+function [aap,resp] = aamod_QA(aap, task)
+%
+% generate a QA summary of functional data
+%
+% Change History
+%
+% fall 2023 [MSJ] -- new
+%
+
+resp='';
+
+switch task
+	
+    case 'report'
+        
+    case 'doit'		
+        
+ 
+        nvoxels = aap.tasklist.currenttask.settings.nvoxels;
```

Comment:
It would be more elegant to use the aas_getsettings function.

Reply:
Will change...